### PR TITLE
505 slow down initial init

### PR DESCRIPTION
**Pull request by devinsag**
Opening PR to have a place for review and feedback.
Please see https://gitlab.com/architect-io/architect-cli/-/issues/505

### CODE REVIEW

File: src/commands/init.ts [hunk 1]
```diff
@@ -62,11 +63,26 @@ export abstract class InitCommand extends BaseCommand {
...
     this.log(`Start the process to create your project '${project_name}'.`);
     const selections = await ProjectUtils.getSelections();
+
+    this.log(`                                                   7###        s##`);
+    this.log(`    ;####^####           ,###Q                      @##        7%"  ]##b                     @##p`);
+    this.log(`   {####  ####           # @##p    W##pa##  ,s#WW#m @##,###m  *### W@##MWm ,sMWWm,   ,m#WW#pW###MWL`);
+    this.log(`  #####      &####      @\`  @##    j##b^7Y ###    ^ @##  '###  @##~ @##b  ###  '##b ###    \` @##b`);
+    this.log(` WWWWQ,,,   #####      @WWWWW###   j##b   j###      @##   ###  @##~ @##b ]###"T"TTb]###      @##b`);
+    this.log(`     @###b ####b      ]#     @##Q  j##b    @##p   , @##   ###  @##~ @##b  @##p    ,^@##    , @##b`);
+    this.log(`     %888W8888"      *WWW    4WWWW %WWW=    7%W#MT *WWWW 4WWW=*WWWW  %W#W^ ^%WWWWb   7%WWW"   %WWW^\n`);
```
Comment (mtgoncalves1) [inline src/commands/init.ts:73]: I really love this logo. My only concern is the case where the user is using a small terminal screen.

File: src/commands/init.ts [hunk 2]
```diff
@@ -62,11 +63,26 @@ export abstract class InitCommand extends BaseCommand {
...
+    this.log('\n✨ And now, for some Architect Magic ✨\n');
     await ProjectUtils.updateArchitectYamls(this.app, selections, project_name);
     await ProjectUtils.linkSelections(this.app, selections, project_name);
-    this.log(`Successfully created project ${project_name}.`);
-    this.log(chalk.blue(`Now try running your project using the instructions in ${project_name}/README.md.`));
+
+    const root_path = path.join(project_name, ProjectUtils.getRootComponent(selections), 'architect.yml');
+    this.log(chalk.grey('# Visit https://docs.architect.io/deployments/local-environments/#local-registration to learn more about local registration\n'));
+    this.log(chalk.green(`Successfully created project ${project_name}.\n`));
+    this.log(`Your App is ready to be deployed by architect!\nJust run:\n\t$ architect dev ${root_path}\n`);
```
[the remark this answers is not in the document]
Comment (devinsag) [reply src/commands/init.ts:85]: my mistake, I was referencing https://gitlab.com/architect-io/architect-cli/-/issues/500 initially when designing the text and missed the changes. I'll go back and make sure it is correct to the updated version.

File: src/commands/init.ts (hunk 1 above)
Comment (devinsag) [reply src/commands/init.ts:73]: I agree. I'll go a head and switch back to the original header

Comment (devinsag) [reply src/commands/init.ts]: updated logo to one more compatible with terminal screens less than 35px by 134px in size (eg. visual studio console integration)

File: src/commands/init.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (devinsag) [reply src/commands/init.ts]: updated display to match updated template

File: src/common/utils/prompt-utils.ts [hunk 3]
```diff
@@ -51,4 +53,25 @@ export default class PromptUtils {
...
+   *     - 'Fetching the list of available application types... Loading...'
+   *     - 'Fetching the list of available application types... DONE'
+   *
+   * @param {string} spinner_prefix - prepended spinner text.
+   * @param {string} spinner_loading_postfix - text appending spinner for amount of time in ms specified in spinner_duration_ms.
+   * @param {string} spinner_completed_postfix - text appending spinner replacing spinner_loading_postfix after spinner_duration_ms has elapsed.
+   * @param {number} spinner_duration_ms - time in ms to display spinner.
+   *
+   */
+  public static async oclif_timed_spinner(spinner_prefix = '', spinner_loading_postfix = '', spinner_completed_postfix: string = chalk.green('✓'),
```
Comment (tjhiggins) [inline src/common/utils/prompt-utils.ts:71]: func should be camelcase

Comment (devinsag) [reply src/common/utils/prompt-utils.ts]: updated to camelcase

**Pull request merged**